src/test/java/picard/util/IntervalListToolsTest.java

```diff
@@ -49,16 +58,124 @@
+    private static final boolean DEFAULT_INVERT_ARG = false;
+    private static final boolean DEFAULT_UNIQUE_ARG = false;
+    private static final IntervalListTools.Action DEFAULT_ACTION = IntervalListTools.Action.INTERSECT;
+    private static final boolean DEFAULT_DONT_MERGE_ABUTTING = false;
+    private static final String INTERVAL_LIST_EXTENSION = ".interval_list"; // Note that '.' is included
+
+
+
+
+    @Test(groups = "cloud")
+    public void tsatoCloudInput() throws IOException {
+        // tsato: this is how Louis configured the cloud test, but we don't seem to need "groups = 'cloud'" stuff
+        // e.g. testCloud() below seems to work.
+    }
+
+    @DataProvider
+    public Object[][] cloudOutputLocation() {
+        final boolean cloud = true;
+        final boolean local = false;
+        return new Object[][]{
+                {cloud},
+                {local}
+        };
+    }
+
+
+    // tsato: The input file is in cloud, the output is local (can I organize this a little better?)
+    @Test(dataProvider = "cloudOutputLocation")
+    public void testCloudInput(boolean outputInCloud) throws IOException {
+        // Run the tool with the local input files and get the "expected" output
+        final IntervalList expectedIntervals = tester(DEFAULT_ACTION, DEFAULT_INVERT_ARG, DEFAULT_UNIQUE_ARG, DEFAULT_DONT_MERGE_ABUTTING, scatterable, secondInput, false);
+        final IntervalList intervalListCloudInput = tester(DEFAULT_ACTION, DEFAULT_INVERT_ARG, DEFAULT_UNIQUE_ARG, DEFAULT_DONT_MERGE_ABUTTING, scatterableCloud.toPath(), secondInputCloud.toPath(), outputInCloud);
+
+        Assert.assertEquals(expectedIntervals.size(), intervalListCloudInput.size());
+
+        for (int i = 0; i < expectedIntervals.size(); i++) {
+            final Interval local = expectedIntervals.getIntervals().get(i);
+            final Interval cloud = intervalListCloudInput.getIntervals().get(i);
+            Assert.assertEquals(local, cloud);
+        }
+    }
+
+
+    private String getExpectedScatteredIntervalList(final int scatterCount) {
+        final List<String> args = buildStandardTesterArguments(DEFAULT_ACTION, DEFAULT_INVERT_ARG, DEFAULT_UNIQUE_ARG, DEFAULT_DONT_MERGE_ABUTTING, scatterableCloud.toPath(), secondInputCloud.toPath());
+        try {
+            final Path expectedOutput = Files.createTempDirectory("scatter_test");
+            args.add("SCATTER_COUNT=" + scatterCount);
+            args.add("OUTPUT=" + expectedOutput.toUri());
+            Assert.assertEquals(runPicardCommandLine(args), 0); // tsato: why does this trigger defaulting to .interval_list
+            return expectedOutput.toUri().getPath();
+        } catch (IOException e){
+            throw new PicardException("Encountered an IO Exception while running IntevalListTools", e);
+        }
+    }
+
+    @Test
+    public void testScatterOutputInCloud3() { // tsato: added 3 because it was running all the picard tests, figured something got misconfigured
```

Comment:
When I remove "3" from the name and run the test (right-click -> run on IntelliJ), it seems to run a bunch of tests (besides this one...)